iOSClient/More/NCMore.swift

```diff
@@ -491,25 +513,9 @@ class NCMore: UIViewController, UITableViewDelegate, UITableViewDataSource {
 }
 
 class CCCellMore: UITableViewCell {
-
```

Comment:
Avoid adding unnecessary new line to avoid code conflicts and our customization.